src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -56,6 +56,8 @@ public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFact
         public override void LogFunctionExecutionEvent(string executionId, string siteName, int concurrency, string functionName,
             string invocationId, string executionStage, long executionTimeSpan, bool success)
         {
+            var logger = _loggerFactory.GetOrCreate(FunctionsExecutionEventsCategory);
+            WriteEvent(logger, $"{executionStage}");
```

Comment:
nit: you probably don't need to use string interpolation here since you're just sending one string anyway. 

```suggestion
            WriteEvent(logger, executionStage);
```